cli/parse_pdfs.py

```diff
@@ -470,10 +516,10 @@ def parse_file(
         )
 
 
-def _pdf_num_pages(file: str):
+def _pdf_num_pages(file: str) -> int:
     """Get the number of pages in a pdf file."""
     try:
-        return fitz.open(file).page_count
+        return fitz.open(file).page_count  # type: ignore
```

Comment:
Believe this is meant to be used like:

     fitz.Document(file).page_count
     
     

From the __init__.py 


      fitz.open = fitz.Document


After using that the type hint goes away (haven't run it.)

Reply:
I'm removing this function anyway, it's not used anymore.